### DISCUSSION TITLE
Bare metal (UPI) not a platform option when creating manifests

### DISCUSSION THREAD

**Discussion by sabman3** [okd-project/okd#1843] [General]
Running openshift-install (openshift-install 4.14.0-0.okd-2023-12-01-225814) to create manifiests and I don't have an option for bare metal. The platforms that show are vSphere and cloud providors. I've seen screen shots showing bare metal as an option online, but it is not available. So how is someone supposed to test using VituallBox or KVM without this option?

**Comment by titou10titou10** [comment]
AFAIK, currently, the only way to do it is to build and customize the assisted-intaller, run and access it via a browser where you'll be able to generate everything for bare metal. I've done it myself some time ago
Ref: https://hackmd.io/tDnHM2BoQru0VgqMGI_Q5g
assisted-installer github: https://github.com/openshift/assisted-service/
But I may be wrong and it may exist another way.

**Comment by peterroth** [comment]
You need a manifest file with your cluster's configuration and run the `openshift-install create manifests --dir={your directory}` and `openshift-install create ignition-configs --dir={your directory}` commands against that. You can find an example file here: [Ross Brigoli's GitHub](https://github.com/rossbrigoli/okd4_files/blob/master/install-config.yaml).
